charts/dugtrio/values.yaml

```diff
@@ -141,6 +141,28 @@ extraPorts: []
 # -- Additional env variables
 extraEnv: []
 
+serviceMonitor:
+  # -- If true, a ServiceMonitor CRD is created for a prometheus operator
+  # https://github.com/coreos/prometheus-operator
+  enabled: true
```

Comment:
Could we flip this to `false` by default to align with the other charts please? This way users who don't have the prometheus CRDs installed are fine with the defaults 🙏